lib/chef/knife/openstack_floating_ip_create.rb

```diff
@@ -0,0 +1,22 @@
+# Author:: Vasundhara Jagdale (<vasundhara.jagdale@clogeny.com>)
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+
+require 'chef/knife/openstack_helpers'
+require 'chef/knife/cloud/openstack_service_options'
+
+class Chef
+  class Knife
+    class Cloud
+      class OpenstackFloatingIpCreate < Command
+        include OpenstackHelpers
+        include OpenstackServiceOptions
+
+        banner 'knife openstack floating_ip create (options)'
```

Comment:
Instead of just `create` it should have an option for `allocate`, if possible. The OpenStack terminology is to `allocate` for getting new IPs.